migrations/datastore/mysql/README.md

```diff
@@ -40,6 +40,14 @@ Note: You must use 2 different databases for `datastore` and `eventstore`. You c
 
 The `synchronizeEvents` attribute in the eventstore section is false by default. Setting it to true means that all events will be tracked in order within the same transaction (helpful for testing locally).
 
+You may also customize your database or eventstore connection by providing a [DSN (Data Source Name)](https://github.com/go-sql-driver/mysql#dsn-data-source-name). If provided, this string is used to open the given database rather using the individual variables, i.e. `user`, `password`, `hostname`.
+
+```yaml
+datastore:
+  mysql:
+    dsn: root:@tcp(127.0.0.1:3306)/warrant
```

Comment:
Can you update the example DSN to include `?parseTime=true` and also add a small note about the `parseTime=true` property being required in the DSN if using MySQL?